src/lib.rs

```diff
@@ -24,7 +24,7 @@
 //! let val = noise.apply(&seed, &[42.0, 37.0, 2.0]);
 //! ```
 
-#![deny(missing_copy_implementations)]
+//#![deny(missing_copy_implementations)]
```

Comment:
It was giving me an error about missing copy implementations on the Checkerboard struct. Fixed it in 9884435.